### PR TITLE
MNT: Re-rendered with conda-smithy 1.3.2

### PR DESCRIPTION
**Pull request by ElDeveloper**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
+
+os: osx
+osx_image: beta-xcode6.1
 
 env:
-  matrix:
-    
-    - CONDA_PY=27
-    - CONDA_PY=34
-    - CONDA_PY=35
```
Comment (ocefpaf) [inline .travis.yml]: @ElDeveloper something is not OK with the builds matrix. Let me make some tests locally before you merge this.

**Comment by jakirkham** [comment]
Please make sure to do a build number bump.

**Comment by ocefpaf** [comment]
@ElDeveloper let's close this in favor of #3 to avoid shipping broken packages.

**Comment by ElDeveloper** [comment]
Sounds good, thanks both!!

File: .travis.yml (hunk 1 above)
Comment (jakirkham) [inline .travis.yml]: The other case where this seems to happen appears to be not having everything come from the `conda-forge` channel (if possible). That's what needed to happen for the re-rendering bot at least (see link below). Though I suspect this covering up the real problem.

xref: https://github.com/conda-forge/conda-forge-maintenance/pull/7

**Pull request closed without merging**